src/components/VmDetails/cards/UtilizationCard/DiskCharts.js

```diff
@@ -43,18 +72,20 @@ const DiskCharts = ({ vm, isRunning, id, ...props }) => {
   })
 
   const { unit, value: disks } = convertValueMap('B', { actualSize, provisionedSize })
-  const used = round(disks.actualSize, 1)
   const available = round(disks.provisionedSize - disks.actualSize, 1)
   const total = round(disks.provisionedSize, 1)
 
   return (
     <UtilizationCard className={style['chart-card']} id={id}>
       <CardTitle>Disk <span style={{ fontSize: '55%', verticalAlign: 'super' }}>(storage allocations)</span></CardTitle>
```

Comment:
We can probably take the super-scripted "storage allocations" off the `CardTitle` now.